patches/cli/0012-Exclude-Localization-in-source-build.patch

```diff
@@ -0,0 +1,25 @@
+From ff5dbadc760046a2557194bae052de8a82d866be Mon Sep 17 00:00:00 2001
+From: dseefeld <dseefeld@microsoft.com>
+Date: Thu, 26 Apr 2018 15:45:05 +0000
+Subject: [PATCH] Exclude Localization in source-build
+
+---
+ src/redist/redist.csproj | 2 +-
+ 1 file changed, 1 insertion(+), 1 deletion(-)
+
+diff --git a/src/redist/redist.csproj b/src/redist/redist.csproj
+index da07741..c2b2a87 100644
+--- a/src/redist/redist.csproj
++++ b/src/redist/redist.csproj
+@@ -19,7 +19,7 @@
+     <PackageReference Include="NuGet.Build.Tasks" Version="$(NuGetBuildTasksPackageVersion)" />
+     <PackageReference Include="Microsoft.TestPlatform.CLI" Version="$(MicrosoftTestPlatformCLIPackageVersion)" />
+     <PackageReference Include="Microsoft.TestPlatform.Build" Version="$(MicrosoftTestPlatformBuildPackageVersion)" />
+-    <PackageReference Include="NuGet.Localization" Version="$(NuGetProjectModelPackageVersion)" />
++    <PackageReference Condition=" '$(DotNetBuildFromSource)' != 'true' " Include="NuGet.Localization" Version="$(NuGetProjectModelPackageVersion)" />
```

Comment:
It looks like this package contains a bunch of string loc resource DLLs. Any idea if CLI commands might look for these translations and fail if we don't include them? Think we need to do a non-English smoke test?

Reply:
I've never tried it before, not sure what you have to do.

Reply:
The locale-named folders in the CLI we download and the preview2 tarball-built CLI are all empty, so it seems normal to me for these not to exist. I'm guessing we're probably fine here, but I filed https://github.com/dotnet/source-build/issues/485 to follow up more generally on localization.

Reply:
My Debian machine has the official dotnet build in German now so I can test this out manually at least.  It looks like switching locales does not work in a Docker container, and you have to remove locale variables from /etc/ssh/sshd_config so they aren't overridden when you log in.